src/authorizations/utils/permissions.ts

```diff
@@ -329,14 +329,20 @@ export const generateDescription = apiPermissions => {
     }
   } else {
     apiPermissions.forEach(perm => {
-      if (perm.resource.name) {
+      if (perm.resource.type === 'orgs') {
         generatedDescription += `${capitalize(perm.action)} ${
           perm.resource.type
-        } ${perm.resource.name} `
+        }  `
       } else {
-        generatedDescription += `${capitalize(perm.action)} ${
-          perm.resource.type
-        } `
+        if (perm.resource.name) {
+          generatedDescription += `${capitalize(perm.action)} ${
+            perm.resource.type
+          } ${perm.resource.name} `
+        } else {
```

Comment:
it might be easier to read without all the extra brackets and syntax if instead of an `else` this is just set as a default case. I personally find it easier to follow, as it generally puts the default case at the top, then follows it with a special case triggered by a conditional

```ts
} else {
  generatedDescription += `${capitalize(perm.action)} ${perm.resource.type} `
  if (perm.resource.name) {
    generatedDescription += `${capitalize(perm.action)} ${
      perm.resource.type
    } ${perm.resource.name} `
  }
}
```